src/api/services/http/dotcms-http-service.ts

```diff
@@ -0,0 +1,45 @@
+import {Http, Response, Request, Headers, RequestOptionsArgs} from '@angular/http'
```

Comment:
I don't the benefit of having this service and core-web-service, I don't think we should create a new one, we should just make a good separation of concerns in core-web-service. 

My 2cents.